sqlite/README.md

```diff
@@ -0,0 +1,364 @@
+firecracker-x86_64 --config-file fc.x86_64.json --api-sock firecracker.socket
+```
+
+The user running the above command must be able to use KVM.
+Typically this means being part of the `kvm` group.
+Otherwise, run the command above as root or prefixed by `sudo`.
+
+### Run on Firecracker/ARM64
+
+To run the Unikraft image:
+
+```console
+rm -f firecracker.socket
+firecracker-aarch64 --config-file fc.arm64.json --api-sock firecracker.socket
+```
+
+The user running the above command must be able to use KVM.
+Typically this means being part of the `kvm` group.
+Otherwise, run the command above as the `root` account or prefixed by `sudo`.
+
+### Run on Xen/x86_64
+
+To run the Unikraft image:
+
+```console
+sudo xl create -c xen.x86_64.cfg
+```
+
+You need use `sudo` or the `root` account to run Xen.
+
+### Run on Xen/ARM64
+
+To run the Unikraft image:
+
+```console
+sudo xl create -c xen.arm64.cfg
+```
+
+You need use `sudo` or the `root` account to run Xen.
+
+## Test
+
+### Close QEMU
+
+To close the QEMU virtual machine, use the `Ctrl+a x` keyboard shortcut;
+that is press the `Ctrl` and `a` keys at the same time and then, separately, press the `x` key.
+
+### Close Firecracker
+
+To close the Firecracker virtual machine, open another console and use the command:
+
+```console
+sudo pkill -f firecracker
+```
+
+### Close Xen
+
+To close the Xen virtual machine, open another console and use the command:
+
+```console
+sudo xl destroy sqlite
+```
+
+## Clean Up
+
+Doing a new configuration, or a new build, may require cleaning up the configuration and build artifacts.
+
+In order to remove the build artifacts, use:
+
+```console
+make clean
+```
+
+In order to remove fetched files also, that is the removal of the `workdir/build/` directory, use:
+
+```console
+make properclean
+```
+
+In order to remove the generated `.config` file as well, use:
+
+```console
+make distclean
+```
+
+## Customize
+
+### Customize the Filesystem Contents
+
+The main way to customize SQLite is to update its configuration in the directory of the filesystem:
+
+```console
+rootfs/
+|-- chinook.db
+`-- script.sql
+```
+
+After updating the filesystem contents, you need to [rebuild the filesystem](#build-the-filesystem) and the [run](#run) the unikernel.
+No rebuild of the unikernel is necessary.
+
+### Use a Different Filesystem Type for QEMU
+
+You can use [`9pfs`](https://github.com/unikraft/unikraft/tree/staging/lib/9pfs) as an alternate filesystem to initrd.
+Note that 9pfs does not work with Firecracker.
+And it requires re-building Xen to add 9pfs support.
+Below find instructions on running SQLite on QEMU with 9pfs support.
+
+You need to use these contents for the [`Config.uk`](Config.uk) configuration file:
+
+```text
+config SQLITE
+bool "Configure SQLite application with 9pfs as rootfs"
+default y
+  select LIBSQLITE
+  select LIBSQLITE_MAIN_FUNCTION
+  select LIBVFSCORE
+  select LIBVFSCORE_AUTOMOUNT_UP
+  select LIBUK9P
+  select LIB9PFS
```

Comment:
Indent lines starting from the 2nd one with 8 blanks. Be consistent with the other apps.